### DISCUSSION TITLE
Curation report - week of Jan 3rd 2022

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/GPML-Repo#35] [Curation reports]
Hi,

here is my curation week report.

Only 12 recently changed pathways, but still around 30 without updated approved tag.

I updated some of the approved tags, but not the Homology Converted tags. Should these be updated as well? I guess they should only be indicating the version that initially came from homology conversion, or should it always be connected to the latest version?

Also, are these tags supposed to be on all homology converted pathways, or only specific ones? Is this process automatically? I saw that https://www.wikipathways.org/index.php/Pathway:WP3937 didn't have this tag. In the description, it mentions that it was converted from human to mouse, the homology mapper comment says it is the other way round. As this is a human pathway, I changed it in the description.

https://www.wikipathways.org/index.php/Pathway:WP2435 has neither the approved, nor the needs work tag (only the fix interactions and annotate nodes tags, but both are quite complete). Is this good enough to be approved?

This COVID-19 pathway was last changed in November: https://www.wikipathways.org/index.php/Pathway:WP5133. Is it ready?

https://www.wikipathways.org/index.php/Pathway:WP5136 and https://www.wikipathways.org/index.php/Pathway:WP5137 have cytochrome P450 as Metabolite with a ChEBI ID. Is there a reason for not having it as protein or gene product? And are these pathways interesting for the lipid maps portal?

https://www.wikipathways.org/index.php/Pathway:WP397 is currently approved, but 200 of 200 interactions are not properly connected, and it has no bibliography (it seems to be a homology converted netpath pathway, but it looks very different to the original https://www.wikipathways.org/index.php/Pathway:WP53).

@egonw, do you still need the test pathway https://www.wikipathways.org/index.php/Pathway:WP5131? It is from September.

Search works.

Next one is Friederike.

Cheers, 
Daniela

**Comment by AlexanderPico** [comment]
Thanks for the great curation work, Daniela! Some responses to your questions...

1. Yes, all approved tags should be updated to latest version or bad edits should be reverted. I followed up on the ones you skipped.
2. WP3937 was originally modeled for mouse. A human version was made using homology mapping. The HomologyMapper comment is correct. I just removed the redundant sentence from the description to avoid confusion.
3. WP2435. Agreed. I added the approved tag.
4. WP5133. Agreed. I added the approved tag.
5. CYP450 is a superfamily of dozens of genes. I agree it's odd, but it looks like these chemical databases have accurate IDs for this enzyme superfamily; whereas NCBI or Ensembl does not. So, I guess it's fine as it is??
6. Most of the NetPath pathways do not have connected interactions. We made an exception for these early on since they provide such rich gene content. The hope was that they'd eventually connect the end points...
